README.md

```diff
@@ -47,13 +52,52 @@ notebook_login()
 ```
 huggingface-cli login
 ```
-Then:
+Then
+
+**With `package_to_hub()`**:
+
+```python
+import gym
+
+from stable_baselines3 import PPO
+from stable_baselines3.common.evaluation import evaluate_policy
```

Comment:
I don't think you need `evaluate_policy` here

Reply:
Thanks!